Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1203,7 +1215,13 @@ respondQuote (driverId, _) req = do
                   waitingTime = Nothing,
                   driverSelectedFare = mbOfferedFare,
                   customerExtraFee = searchTry.customerExtraFee,
-                  nightShiftCharge = Nothing
+                  nightShiftCharge = Nothing, -- FIX ME: Params below me are for rentals
+                  rideStartTime = Nothing,
```

Comment:
Why all these extra params??

Reply:
These are for fare calculation of Rental rides